modules/ipi-install-configuring-host-network-interfaces-in-the-install-config.yaml-file.adoc

```diff
@@ -6,64 +6,140 @@
++
+[source,terminal]
+----
+    hosts:
+      - name: openshift-master-0
+        role: master
+        bmc:
+          address: redfish+http://<out_of_band_ip>/redfish/v1/Systems/
+          username: <user>
+          password: <password>
+          disableCertificateVerification: null
+        bootMACAddress: <NIC1_mac_address>
+        bootMode: UEFI
+        rootDeviceHints:
+          deviceName: "/dev/sda"
+        networkConfig: <1>
+          interfaces:
+          - name: <NIC1_name>
+            type: ethernet
+            state: up
+            ipv4:
+              address:
+              - ip: <IP_address>
+                prefix-length: 24
+              enabled: true
+          dns-resolver:
+            config:
+              server:
+              - <DNS_IP_address>
+          routes:
+            config:
+            - destination: 0.0.0.0/0
+              next-hop-address: <next_hop_IP_address>
+              next-hop-interface: <next_hop_NIC1_name>
+----
+<1> Add NMState YAML syntax to configure host interfaces.
++
 [IMPORTANT]
 ====
 Once deployed, you cannot modify the `networkConfig` configuration setting of `install-config.yaml` file to make changes to the host network interface. Use the Kubernetes NMState Operator to make changes to the host network interface after deployment.
 ====
+
+. Optional: In the case of deploying {product-title} without a DHCP server on the `baremetal` network, use the following procedure to configure a static IP address for the bootstrap VM using Ignition:
+
+.. Create the `bootstrap_config.sh` file:
++
+[source,terminal]
+----
+#!/bin/bash
+
+BOOTSTRAP_CONFIG="[connection]
+type=ethernet
+interface-name=<interface_name>
+[ethernet]
+[ipv4]
+method=manual
+addresses=<ip_address>/<CIDR>
+gateway=<gateway_IP_address>
+dns=<dns_IP_address>"
+
+cat <<_EOF_ > bootstrap_network_config.ign
+{
+  "path": "/etc/NetworkManager/system-connections/<interface_name>.nmconnection",
+  "mode": 384,
+  "contents": {
+    "source": "data:text/plain;charset=utf-8;base64,$(echo "${BOOTSTRAP_CONFIG}" | base64 -w 0)"
+  }
+}
+_EOF_
+
+mv clusterconfigs/bootstrap.ign clusterconfigs/bootstrap.ign.orig
+
+jq '.storage.files += $input' clusterconfigs/bootstrap.ign.orig --slurpfile input bootstrap_network_config.ign > clusterconfigs/bootstrap.ign
+----
++
+Replace the two `<interface_name>` instances with the name of the NIC. Replace `<ip_address>` and `<CIDR>` with the IP address and CIDR of the address range. Replace `<gateway_IP_address>` with the IP address of the gateway on the `baremetal` network. Replace `<dns_IP_address>` with the IP address of the DNS server on the `baremetal` network.
+
+.. Make the `bootstrap_config.sh` file executable:
++
+[source,terminal]
+----
+$ chmod 755 bootstrap_config.sh
+----
+
+.. Run the `bootstrap_config.sh` script to create the `bootstrap_network_config.ign` file:
++
+[source,terminal]
+----
+$ ./bootstrap_config.sh
+----
```

Comment:
You could consider including an example output to illustrate what the script outputs when run successfully.

Reply:
There is no output when the script runs successfully.